cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/handlers/NGSIRestHandler.java

```diff
@@ -393,7 +393,7 @@ public void configure(Context context) {
      * @return True is the header value length is wrong, otherwise false
      */
     protected boolean wrongContentType(String headerValue) {
-        return !headerValue.contains("application/json; charset=utf-8");
+        return !headerValue.toLowerCase(Locale.ENGLISH).contains("application/json; charset=utf-8");
```

Comment:
Please add a line in CHANGES_NEXT_RELEASE about this fix (citing issue number #1609)

Reply:
Fixed in b3854cf28c90507a80a511dc0fc1ddd6a9218a40